src/Particles/babylon.solidParticleSystem.ts

```diff
@@ -2,53 +2,53 @@ module BABYLON {
 
     export class SolidParticleSystem implements IDisposable {
         // public members  
-        public particles = new Array<SolidParticle>();
-        public nbParticles = 0;
-        public billboard = false;
-        public counter = 0;
+        public particles: SolidParticle[] = new Array<SolidParticle>();
```

Comment:
I did it because I plan to try a test with asm.js later and having the types will be easier to convert the numerical ones to the right low level types